wram.asm

```diff
@@ -445,58 +445,52 @@ wAttackMissed::
 
 wPlayerSubStatus1::
 ; bit
-; 7 in love
-; 6 rollout
-; 5 endure
-; 4 perish song
-; 3 identified
-; 2 protect
-; 1 curse
 ; 0 nightmare
+; 1 curse
```

Comment:
Maybe this should be handled in a different PR, but why document the bits here when they're documented in `constants/` and associated with the proper label in a comment? Maybe this is one of those places where `EXPORT` is a good idea?

Reply:
Yeah, I'd rather just delete those redundant comments in wram.asm. The `SUBSTATUS_*` constant definitions already comment that they apply to `wPlayerSubStatus1`, `wPlayerSubStatus2`, etc, and have descriptive names. Maybe this wram.asm comment could just be "`; takes a SUBSTATUS_*` value`".

(Also, definitely save that for a separate PR, because I think the constants should also be `SUBSTATUS1_*`, `SUBSTATUS2_*`, etc, to make it clear which go where.)